source/docs/software/wpilib-tools/smartdashboard/testing-commands.rst

```diff
@@ -0,0 +1,25 @@
+Testing commands
```

Comment:
There seems to be a lot of overlap between the Testing Commands article and the Displaying status of commands and subsystems articles

Reply:
Can the [Testing Commands](https://frc-docs--687.org.readthedocs.build/en/687/docs/software/wpilib-tools/smartdashboard/testing-commands.html) article be removed in favor of the **Displaying status of commands and subsystems** article?

Reply:
I'm going to say yes

Reply:
Thanks, removed in fdaf7c4. Ready for a review.